Error importing @langchain/anthropic

### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

```typescript
import { ChatAnthropic } from '@langchain/anthropic';
```

### Error Message and Stack Trace (if applicable)

```bash
$ yarn build
node_modules/@langchain/anthropic/dist/chat_models.d.ts:10:44 - error TS7016: Could not find a declaration file for module '@anthropic-ai/sdk/resources/index.mjs'. '/home/XXX/XXX/node_modules/@anthropic-ai/sdk/resources/index.mjs' implicitly has an 'any' type.
  If the '@anthropic-ai/sdk' package actually exposes this module, try adding a new declaration (.d.ts) file containing `declare module '@anthropic-ai/sdk/resources/index.mjs';`

10 import type { Tool as AnthropicTool } from "@anthropic-ai/sdk/resources/index.mjs";
                                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@langchain/anthropic/dist/types.d.ts:2:44 - error TS7016: Could not find a declaration file for module '@anthropic-ai/sdk/resources/index.mjs'. '/home/XXX/XXX//node_modules/@anthropic-ai/sdk/resources/index.mjs' implicitly has an 'any' type.
  If the '@anthropic-ai/sdk' package actually exposes this module, try adding a new declaration (.d.ts) file containing `declare module '@anthropic-ai/sdk/resources/index.mjs';`

2 import type { Tool as AnthropicTool } from "@anthropic-ai/sdk/resources/index.mjs";
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors in 2 files.

Errors  Files
     1  node_modules/@langchain/anthropic/dist/chat_models.d.ts:10
     1  node_modules/@langchain/anthropic/dist/types.d.ts:2
```


### Description

Build an application using `@langchain/anthropic`

### System Info

```bash
$ jlpm info @langchain/anthropic
└─ @langchain/anthropic@npm:0.3.8
   ├─ Instances: 1
   ├─ Version: 0.3.8
   │
   └─ Dependencies
      ├─ @anthropic-ai/sdk@npm:^0.27.3 → npm:0.27.3
      ├─ fast-xml-parser@npm:^4.4.1 → npm:4.5.0
      ├─ zod-to-json-schema@npm:^3.22.4 → npm:3.23.5
      └─ zod@npm:^3.22.4 → npm:3.23.8
```
Platform: **ubuntu 24.04**
Node version: **v20.18.0**
yarn version: **3.5.0**

--- a/libs/langchain-anthropic/src/chat_models.ts
+++ b/libs/langchain-anthropic/src/chat_models.ts
@@ -28,7 +28,7 @@ import { z } from "zod";
 import type {
   MessageCreateParams,
   Tool as AnthropicTool,
-} from "@anthropic-ai/sdk/resources/index.mjs";
+} from "@anthropic-ai/sdk/resources/messages";
 
 import { isLangChainTool } from "@langchain/core/utils/function_calling";
 import { AnthropicToolsOutputParser } from "./output_parsers.js";

--- a/libs/langchain-anthropic/src/types.ts
+++ b/libs/langchain-anthropic/src/types.ts
@@ -1,5 +1,5 @@
 import Anthropic from "@anthropic-ai/sdk";
-import type { Tool as AnthropicTool } from "@anthropic-ai/sdk/resources/index.mjs";
+import type { Tool as AnthropicTool } from "@anthropic-ai/sdk/resources/messages";
 import { BindToolsInput } from "@langchain/core/language_models/chat_models";
 
 export type AnthropicToolResponse = {

